clientv3/watch.go

```diff
@@ -194,6 +199,15 @@ func (w *watcher) Watch(ctx context.Context, key string, opts ...OpOption) Watch
 		}
 	}
 
+	select {
```

Comment:
To receive error from the error channel, because `openWatchClient()` sends the `ErrConnClosed` to that channel. Any opinion?

Thanks.